WordPress/src/main/java/org/wordpress/android/util/config/JetpackProviderSyncFeatureConfig.kt

```diff
@@ -14,6 +14,8 @@ class JetpackProviderSyncFeatureConfig
         BuildConfig.JETPACK_PROVIDER_SYNC,
         JETPACK_PROVIDER_SYNC_REMOTE_FIELD
 ) {
+    override fun isEnabled() = true
```

Comment:
Should we merge it to `trunk` with it's value set to `true`?

Reply:
No, definitely not! :sweat_smile:  We should revert this before merging, or else we'd not only lose our "kill switch", but we'd be hard-coding as enabling the feature outright. I should have mentioned this in the description that this should be reverted before merging (it's only for testing, along with the other hard-coded flags).